### DISCUSSION TITLE
Whenever use wire:navigate scroll position stays same from last page.

### DISCUSSION THREAD

**Discussion by orkuncoding** [livewire/livewire#6779] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.2

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

When I use wire:navigate. Scroll position stays same from last page. I checked documention and issues but i did not anything about that. I think It should start from the top of the page when user navigate. I wrote simple javascript for solution but Is there any option to this default?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
